criu/proc_parse.c

```diff
@@ -1084,6 +1084,20 @@ int parse_pid_status(pid_t pid, struct seize_task_status *ss, void *data)
 			continue;
 		}
 
+		if (!strncmp(str, "NSpid:", 6)) {
```

Comment:
This is a first use of NSpid feature in CRIU. But this is quiet a "new" kernel feature and we use it without compat check. Does it mean that we don't support kernel older than 4.1?

```
git describe --contains e4bc33245124d
v4.1-rc1~115^2~41
```

In VZ criu we do something like this https://src.openvz.org/projects/OVZ/repos/criu/commits/94f4653f20#criu/kerndat.c for compatibility.

Alternatively we can probably use gettid() via parasite (like in dump_one_task->dump_task_threads ... ->dump_thread).

And anyway, should not this hunk be merged in the original patch or be a separate preparation patch instead of fixup like we have it now.

Reply:
In personal talk with @avagin we've decided that requiring NSpid feature from kernel is ok and adding ns support as a later patch is also ok. So it's merged.